node/src/database/db_migrations.rs

```diff
@@ -2031,4 +2210,159 @@ mod tests {
+            &["attempt", "integer", "not", "null"],
+            &["process_error", "text", "null"],
+        ];
+        assert_create_table_stm_contains_all_parts(&*conn, "pending_payable", expected_key_words);
+        assert_table_created_as_strict(&*conn, "receivable");
+        let expected_key_words: &[&[&str]] = &[
+            &["wallet_address", "text", "primary", "key"],
+            &["balance_high_b", "integer", "not", "null"],
+            &["balance_low_b", "integer", "not", "null"],
+            &["last_received_timestamp", "integer", "not", "null"],
+        ];
+        assert_create_table_stm_contains_all_parts(conn.as_ref(), "receivable", expected_key_words);
+        ["payable", "receivable", "pending_payable"]
+            .iter()
+            .for_each(|table_name| {
+                assert_table_does_not_exist(&*conn, &format!("_{}_old", table_name))
+            });
+        assert_values_still_there_plus_pair_of_integer_columns(&*conn, "payable", |row| {
+            assert_eq!(
+                row.get::<usize, Wallet>(0).unwrap(),
+                Wallet::from_str("0xD7d1b2cF58f6500c7CB22fCA42B8512d06813a03").unwrap()
+            );
+            assert_eq!(row.get::<usize, i64>(1).unwrap(), 6156); //high bytes
+            assert_eq!(row.get::<usize, i64>(2).unwrap(), 5467226021000125952); //low bytes
+            assert_eq!(row.get::<usize, i64>(3).unwrap(), 11111);
+            assert_eq!(row.get::<usize, Option<i64>>(4).unwrap(), None);
+            Ok(())
+        });
+        assert_values_still_there_plus_pair_of_integer_columns(&*conn, "receivable", |row| {
+            assert_eq!(
+                row.get::<usize, Wallet>(0).unwrap(),
+                Wallet::from_str("0xD2d1b2eF58f6500c7ae22fCA42B8512d06813a03").unwrap()
+            );
+            assert_eq!(row.get::<usize, i64>(1).unwrap(), -1); //high bytes
+            assert_eq!(row.get::<usize, i64>(2).unwrap(), 9223315252854775808); //low bytes
+            assert_eq!(row.get::<usize, i64>(3).unwrap(), 22222);
+            Ok(())
+        });
+        assert_values_still_there_plus_pair_of_integer_columns(&*conn, "pending_payable", |row| {
+            assert_eq!(
+                row.get::<usize, String>(0).unwrap(),
+                "0xb5c8bd9430b6cc87a0e2fe110ece6bf527fa4f222a4bc8cd032f768fc5219838".to_string()
+            );
+            assert_eq!(row.get::<usize, i64>(1).unwrap(), 0); //high bytes
+            assert_eq!(row.get::<usize, i64>(2).unwrap(), 9123000000000); //low bytes
+            assert_eq!(row.get::<usize, i64>(3).unwrap(), 33333);
+            assert_eq!(row.get::<usize, i64>(4).unwrap(), 1);
+            assert_eq!(row.get::<usize, Option<String>>(5).unwrap(), None);
+            Ok(())
+        });
+        let (rate_pack, encrypted) = retrieve_config_row(&*conn, "rate_pack");
+        assert_ne!(old_rate_pack_in_gwei, DEFAULT_RATE_PACK.to_string());
+        assert_eq!(rate_pack, Some(DEFAULT_RATE_PACK.to_string()));
```

Comment:
Again, it might be better to convert the old rate pack than to overwrite it with the default rate pack.